cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -98,14 +98,16 @@ pub type LocationToAccountId = (
 	AccountId32Aliases<RelayNetwork, AccountId>,
 	// Foreign locations alias into accounts according to a hash of their standard description.
 	HashedDescription<AccountId, DescribeFamily<DescribeAllTerminal>>,
-	// Different global consensus parachain sovereign account.
-	// (Used for over-bridge transfers and reserve processing)
-	GlobalConsensusParachainConvertsFor<UniversalLocation, AccountId>,
-	// Ethereum contract sovereign account.
-	// (Used to get convert ethereum contract locations to sovereign account)
+	GlobalLocationToAccount<UniversalLocation, AccountId>,
+);
+
+pub type GlobalLocationToAccount<UniversalLocation, AccountId> = (
+    GlobalConsensusParachainConvertsFor<UniversalLocation, AccountId>,
 	EthereumLocationsConverterFor<AccountId>,
+    ExternalConsensusLocationsConverterFor<UniversalLocation, AccountId>,
 );
```

Comment:
No need anymore for explicit `GlobalConsensusParachainConvertsFor` and `EthereumLocationsConverterFor`, `ExternalConsensusLocationsConverterFor` already covers them

```suggestion
	ExternalConsensusLocationsConverterFor<UniversalLocation, AccountId>,
);
```

(same for asset-hub-rococo)